src/daisy_seed.cpp

```diff
@@ -207,10 +207,9 @@ void DaisySeed::SetTestPoint(bool state)
 
 void DaisySeed::ConfigureSdram()
 {
-    dsy_gpio_pin *pin_group;
-    sdram_handle.state             = DSY_SDRAM_STATE_ENABLE;
-    pin_group                      = sdram_handle.pin_config;
-    pin_group[DSY_SDRAM_PIN_SDNWE] = dsy_pin(DSY_GPIOH, 5);
+    sdram_config.state = SdramHandle::State::ENABLE;
+    sdram_config.pin_config[(int)SdramHandle::Pin::SDNWE]
```

Comment:
For stronger type safety I would suggest to have each pin as a separate member in the pin_config struct, so that you can leave out the array indexing like this:
```cpp
sdram_config.pin_config.sdnwe = dsy_pin(DSY_GPIOH, 5);
```